supporting_scripts/course-setup-quickstart/README.md

```diff
@@ -0,0 +1,97 @@
+# Create local course and users
+
+Scripts in this folder help to configure and setup your first Artemis course to get started quickly.
+
+## Setup
+
+1. Install the [Python Plugin for IntelliJ](https://plugins.jetbrains.com/plugin/631-python)
+
+2. Enable Python support in your IntelliJ
+
+   `File > Project Structure > Facets > Add - Python` (press the plus and add a Python environment there,
+   make sure that you have configured the python interpreter)
+
+   ![IntelliJ Python Facet Configuration](./images/facets-config.png)
+
+   If not added automatically by IntelliJ add a Module:
+   `File > Project Structure > Modules > Add - Python`
+
+   ![IntelliJ Module Configuration](./images/module-config.png)
+
+3. Configure the `Artemis.main` module for the run configuration of the run
```

Comment:
This should probably be done in the run configuration itself so that no dev has to reconfigure the run definition.

Reply:
I agree, I removed the step from the ReadMe and pushed the adjusted run configuration!

For the other run configurations no changes seem to be needed _(at least the server started with my setup)_